frontend/src/components/filter/FilterMenu.js

```diff
@@ -9,6 +9,7 @@ import FilterItem from './FilterItem';
 import usePrevious from '../../hooks/usePrevious';
 import { filterProp, filterConfigProp } from './props';
 import FilterErrorContext from './FilterErrorContext';
+import './filterMenu.css';
 
```

Comment:
_incredibly_ small thing but we usually capitalize to match react component

Reply:
Name has been updated.